optuna/visualization/_pareto_front.py

```diff
@@ -1,13 +1,11 @@
+from __future__ import annotations
+
 import collections
 import json
 from typing import Any
 from typing import Callable
-from typing import Dict
-from typing import List
 from typing import NamedTuple
-from typing import Optional
 from typing import Sequence
-from typing import Tuple
 from typing import Union
```

Comment:
```suggestion
```
Is it possible to remove this line and `Union` in the 385th line?

Reply:
I think it's possible :)